web/thesauruses/_meta/facts.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Since `facts.json` and `rules.json` are so small, and they don't really apply to a lot of languages yet, can we combine them? Like "Rules and Facts" (`rules_facts.json`) and we just have separate categories for Rules (containing defining and undefining) and Facts (assertions and retractions). And for languages that don't support these can just "not implemented" the whole file really easily.  Thoughts on this?

Reply:
Works for me